app/forms.py

```diff
@@ -57,7 +57,12 @@ class NewPasswordForm(FlaskForm):
         'Repeat Password', validators=[DataRequired(), EqualTo('password')])
     submit = SubmitField('Update Password')
 
-class CreateQnForm(FlaskForm):
+class CreateClass(FlaskForm):
+    className = StringField('Class Name', validators=[DataRequired()])
+    submit = SubmitField('Create Class')
+
+
+class CreateQuiz(FlaskForm):
```

Comment:
Shouldn't there be a difference between a quiz and a question? See the database schema.

Reply:
So based on the above response the form will be able to do "both"

Reply:
Since the Class and Quiz forms are basically the same, you could in fact make a single form that could just have the field `name`. Unless one needs to be more specialized this would make it easier to manage.